src/diffusers/models/transformer.py

```diff
@@ -0,0 +1,102 @@
+from dataclasses import dataclass
+
+import torch
+import torch.nn as nn
+
+from ..configuration_utils import ConfigMixin, register_to_config
+from ..modeling_utils import ModelMixin
+from ..utils import BaseOutput
+from .attention import Block
+
+
+@dataclass
+class MaskedLMOutput(BaseOutput):
```

Comment:
I don't think we want "Language model" outputs in a image generation-only library